kyaml/fn/framework/processors.go

```diff
@@ -355,8 +374,7 @@ func TemplatesFromFile(files ...string) TemplatesFunc {
 	return func() ([]*template.Template, error) {
 		var templates []*template.Template
 		for i := range files {
-			n := filepath.Base(files[i])
-			t, err := template.New(n).ParseFiles(files[i])
+			t, err := parseTemplate(files[i])
```

Comment:
This is not really related. It fixes `TemplatesFromFiles` to use pkger just like `TemplatesFromDir` does. Ultimately we should replace pkger with the Go 1.16 fanciness, but that's for another day.